pandas/core/arrays/sparse.py

```diff
@@ -2230,8 +2243,24 @@ def density(self) -> float:
         Ratio of non-sparse points to total (dense) data points
         represented in the DataFrame.
         """
+        self._validate_sparse()
         return np.mean([column.array.density for _, column in self._parent.items()])
 
+    def is_sparse(self) -> "Series":
+        """
+        Returns a boolean Series specifying if each column
+        of the DataFrame is sparse
+
+        .. versionadded:: 1.0.0
+
+         Returns
```

Comment:
too many spaces here?

Reply:
(horizontal ones, it doesn't align with the lines above)